package.json

```diff
@@ -60,7 +60,7 @@
     "@datapunt/amsterdam-react-maps": "0.1.2-alpha.1",
     "@datapunt/asc-assets": "^0.18.1",
     "@datapunt/asc-core": "^0.18.1",
```

Comment:
shouldn't this be removed?

Reply:
`asc-core` is still used in the application